src/plugins/console/public/application/models/legacy_core_editor/legacy_core_editor.ts

```diff
@@ -412,7 +412,30 @@ export class LegacyCoreEditor implements CoreEditor {
             lineNumber: pos.row + 1,
             column: pos.column + 1,
           };
-          autocompleter(position, prefix, callback);
+
+          const getAnnotationControls = () => {
+            let customAnnotation: Annotation;
+            return {
+              setAnnotation(text: string) {
+                const annotations = DO_NOT_USE_2.getAnnotations();
```

Comment:
Haha yeah, this is part of a pattern I started to keep Ace-specific concepts outside of the autocomplete system so that we could migrate to a newer editor, like Monaco. We could probably rename the variable to describe what it actual is: `aceEditorSession` or something. And a comment that we should not be passing this to the autocompleter